src/bankAccount.js

```diff
@@ -1,7 +1,28 @@
 'use strict';
 
-function BankAccount() {
-  // Write your code here
+function BankAccount(name, money) {
+  this.name = name;
+  this.money = money;
+  this.transactions = [];
+  this.transactions.push(`Initial: ${this.money}`);
```

Comment:
create an array with this value, no need to push right after you created an empty one